framework/src/main/java/org/checkerframework/framework/flow/CFAbstractStore.java

```diff
@@ -263,47 +269,42 @@ public void updateForMethodCall(
                     FieldAccess fieldAccess = e.getKey();
```

Comment:
Can you reduce the complexity of this method, e.g. by moving this whole else branch on line 265 above into a helper method?